components/TopNav.vue

```diff
@@ -1,40 +1,52 @@
 <template>
     <div
-        class="p-4 pl-8 border border-secondary-bg/20 flex justify-between items-center bg-gradient-to-t from-secondary-bg/30 via-primary-bg to-primary-bg">
-        <div data-testid="desktop-nav" v-if="$viewport.isGreaterThan('desktop')" class="flex items-center">
-            <div class="font-semibold text-xl group transition-colors items-start">
-                <NuxtLink class="flex" to="/">
-                    <SVGSiteLogo role="img" title="site icon"
-                        class="mr-1 w-10 h-10 flex-shrink-0 align-middle fill-primary group-hover:fill-primary-hover" />
-                    <div class="title-text flex flex-col flex-shrink-0" data-testid="logo">
-                        <div class="text-lg text-primary group-hover:text-primary-hover">
-                            Find a Doc
-                        </div>
-                        <div class="text-sm text-primary leading-none group-hover:text-primary-hover">
-                            Japan
-                        </div>
+        class="p-4 pl-5 border border-secondary-bg/20 flex justify-between items-center bg-gradient-to-t from-secondary-bg/30 via-primary-bg to-primary-bg">
+        <div id="mobile-site-icon"
+            class="landscape:hidden font-semibold text-xl group transition-colors flex justify-between items-start">
+            <SVGSiteLogo role="img" title="site icon"
+                class="mt-1 mr-1 w-10 h-10 flex-shrink-0 align-middle fill-primary group-hover:fill-primary-hover" />
+            <div class="title-text flex flex-col flex-shrink-0" data-testid="portrait-logo">
+                <div class="text-lg text-primary group-hover:text-primary-hover">
+                    Find a Doc,
```

Comment:
♻️

Reply:
I don't think this one should be as it's part of a visual logo and it is split into parts.
It also doesn't change with translations